brainscore_vision/models/crossvit_pretrained/test.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+from pytest import approx
+
+from brainscore_vision import score
+from brainscore_vision.utils import seed_everything
+seed_everything(42)
+
+
+@pytest.mark.private_access
+@pytest.mark.memory_intense
+@pytest.mark.parametrize("model_identifier, benchmark_identifier, expected_score", [
+    ("cv_18_dagger_408_pretrained",
+     "MajajHong2015public.IT-pls", approx(0.5362, abs=0.0005)),
+])
+def test_score(model_identifier, benchmark_identifier, expected_score):
+    actual_score = score(model_identifier=model_identifier, benchmark_identifier=benchmark_identifier,
+                         conda_active=True)
```

Comment:
I can see having to set this flag every time being confusing for users. @kvfairchild can we simplify this?